salt/states/boto_secgroup.py

```diff
@@ -379,7 +379,7 @@ def _get_rule_changes(rules, _rules):
     return (to_delete, to_create)
 
 
-def _rules_present(name, rules, vpc_id=None, vpc_name=None,
+def _rules_present(name, rules=None, vpc_id=None, vpc_name=None,
```

Comment:
What is the reasoning behind changing the `rules` arg to a kwarg with a default of `None`?